reportportal_client/service.py

```diff
@@ -191,7 +192,7 @@ def __init__(self,
         self.is_skipped_an_issue = is_skipped_an_issue
         self.base_url_v1 = uri_join(self.endpoint, "api/v1", self.project)
         self.base_url_v2 = uri_join(self.endpoint, "api/v2", self.project)
-        self.max_pool_size = 50
+        self.max_pool_size = max_pool_size
```

Comment:
You can use max_pool_size directly doing session.mount below without creating an instance attribute.